controller/src/main/java/org/jboss/as/controller/CompositeOperationHandler.java

```diff
@@ -113,6 +113,7 @@ public void handleRollback(OperationContext context, ModelNode operation) {
                     ModelNode stepResponse = responseMap.get(stepName);
                     if (stepResponse.hasDefined(FAILURE_DESCRIPTION)) {
                         failureMsg.get(ControllerLogger.ROOT_LOGGER.compositeOperationFailed(), ControllerLogger.ROOT_LOGGER.operation(stepName)).set(stepResponse.get(FAILURE_DESCRIPTION));
+                        failureMsg.get(ControllerLogger.ROOT_LOGGER.compositeOperationFailed(), "Operation: " + ControllerLogger.ROOT_LOGGER.operation(stepName)).set(operationMap.get(stepName));
```

Comment:
Is this change necessary to resolve WFCORE-2337? If not I'd prefer to deal with it separately. We'd need i18n of "Operation:", and I'm not sure whether the output here is right.